flybywiresim/a32nx
# fix(ecam): Hide REFUELG immediately once refueling is complete

Fixes #5072 

## Summary of Changes
Previosly, the REFUELG message would stay on the ECAM if one started the refueling process on the EFB but then changed to a different page during the refueling process. The only way to make the message disappear was to go to the Fuel page on the EFB. This has been fixed now.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): BlueberryKing#6641

## Testing instructions
- Spawn in cold & dark. There should be about 9.5t of fuel on board.
- Start the refueling process from the EFB, but switch to a different page such as Dashboard.
- Monitor fuel level on the ECAM. Ensure the REFUELG message disappears once the desired fuel level has been reached.
- Try this with different refueling speeds as well as defueling


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
